ui/src/components/retreat/home/HomeContainer.js

```diff
@@ -39,5 +39,4 @@ const RetreatHomeContainer = () => {
     </Box>
```

Comment:
user should exist in the props if you add it to RetreatHomeContainer in line 10
```
const RetreatHomeContainer = (props) => {
```

you should be able to see what it looks like if you just console.log(props)
you will then have to pass this all the way to your praise wall